src/pe_source/dnstwistscript.py

```diff
@@ -0,0 +1,269 @@
+"""Use dnstwist to fuzz domain names and cross check with a blocklist."""
+# Standard Python Libraries
+import datetime
+import json
+import pathlib
+import traceback
+
+# Third-Party Libraries
+import dnstwist
+import dshield
+import psycopg2.extras as extras
+import requests
+
+# cisagov Libraries
+from pe_reports import app
+
+from .data.pe_db.db_query_source import (
+    addSubdomain,
+    connect,
+    getDataSource,
+    getSubdomain,
+    org_root_domains,
+    query_orgs_rev,
+)
+
+date = datetime.datetime.now().strftime("%Y-%m-%d")
+LOGGER = app.config["LOGGER"]
+blocklistStatus = True
+
+
+def checkBlocklist(dom, sub_domain_uid, source_uid, pe_org_uid, perm_list):
+    """Cross reference the dnstwist results with DShield blocklist."""
+    malicious = False
+    attacks = 0
+    reports = 0
+    global blocklistStatus
+    if "original" in dom["fuzzer"] or "dns_a" not in dom:
+        return None, perm_list
+    else:
+        domain_a = str(dom["dns_a"][0])
+        if domain_a == "!ServFail":
+            return None, perm_list
+        LOGGER.info(domain_a)
+
+        # Making api request to blocklist.de, if timeout is reached, set blocklistStatus to False and not check it again
+        try:
+            if blocklistStatus:
+                response = requests.get(
+                    "http://api.blocklist.de/api.php?ip=" + domain_a,
+                    timeout=10,
+                ).content
+                if str(response) != "b'attacks: 0<br />reports: 0<br />'":
+                    malicious = True
+                    attacks = int(str(response).split("attacks: ")[1].split("<")[0])
+                    reports = int(str(response).split("reports: ")[1].split("<")[0])
+        except requests.exceptions.Timeout:
+            blocklistStatus = False
+            LOGGER.info("Timeout on blocklist.de API")
+
+        # Check IP in DShield API
+        try:
+            results = json.loads(
+                dshield.ip(str(dom["dns_a"][0]), return_format=dshield.JSON)
```

Comment:
Not quite sure why this change (originally made as part of https://github.com/cisagov/pe-reports/pull/259/commits/66b3c334430294fc4e60ed62f82b27f309cf5f08) was undone in https://github.com/cisagov/pe-reports/pull/259/commits/ab4c34c46e6c97731f805a91b43d3b451861d450:
```suggestion
                dshield.ip(domain_a, return_format=dshield.JSON)
```